ejb-security-programmatic-auth/src/main/java/webapp/WEB-INF/beans.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not sure we need this, you are not adding any CDI injection

Reply:
removed it